applications/home/views.py

```diff
@@ -160,9 +160,9 @@ def forgot_password_page(request):
 
 		if password_reset_code is not None:
 			users.send_password_reset_email(request, password_reset_code)
-			context["success_message"] = "Email has been sent!"
+			context["success_message"] = "You will receive an email with link to update the password!"
 		else:
-			context["error_message"] = "Email does not exist!"
+			context["error_message"] = "You will receive an email with link to update the password!"
```

Comment:
Is the else clause supposed to apply when the user wants to reset the password but the password_reset_code is not set? When does this happen? Since both messages are the same, do we need an if-then-else statement?

I also suggest changing the message to "You will receive an email message with a link to update your password."

Reply:
@tmmurali we need if-then-else condition to check if the reset code is set or not and if we should send an email. I think we can get rid of the error_message though. We can set `message` field instead of `success_message` or `error_message`.